app/helpers/conditional_partials_helper.rb

```diff
@@ -35,6 +35,7 @@ def render_conditional_partials(partials_list_name, options={}, &block)
   # @param [Symbol] partials_list_name
   # @return [Boolean]
   def will_partial_list_render? partials_list_name, *args
+    return false unless controller.class.respond_to?(:conditional_partials)
```

Comment:
This fixes a bug introduced in a previous PR (I had previously moved the share links to make them more noticeable)